src/components/Header.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import SunflowerLogo from "../sunflowerlogo.png";
+import {ReactComponent as Hamburger} from "../hamburger.svg";
+
+const Header = () => {
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  const handleClick = (event: any) => {
+    setAnchorEl(event.currentTarget);
+  };
+
```

Comment:
Please avoid `any` types, because then theres no point of using typescript
Material UI has Typescript sample code so please use it as reference
https://material-ui.com/components/menus/#SimpleMenu.tsx

Reply:
Understood